### ISSUE TITLE
[Bug] One or more wems fails to convert 

### ISSUE THREAD

**Issue by k-popfilters** [vgmstream/vgmstream#1251]
hi, i work with wem files and all of the convert flawlessly but one or more will refuse to convert...  i hope the team is aware of this bug and fix it soon! thank you for making such an awesome tool!
![just two dont show duration](https://user-images.githubusercontent.com/105951186/197407286-8dc52e7f-cd61-43aa-bc1f-aa92f747daba.PNG)

**Comment by nblockbuster** [comment]
[This zip](https://github.com/vgmstream/vgmstream/files/9847182/LoadingScreen.zip) includes some of the broken ones, they're from overwatch 2, and vgmstream exits with `failed opening`

**Comment by bnnm** [comment]
Fixed in #1253 (see https://vgmstream.org/downloads/, may take a while)

You may need to set the "downmix" option or use txtp to remove channels as those use 12 channels, though half are silent somehow.

**Comment by bnnm** [comment]
I was under the impression last PR was meant to fix this, oh well.

I'll recheck later AK's code since they have a function that sets coupleds based on channel layout when mapping is 255. Incidentally mapping 255 also sets changes output channel order to wave style (unlike Opus's default vorbis style).

**Comment by bnnm** [comment]
Rechecked those files and actually seem correct. After testing all possible coupleds, only 0 outputs correctly, and AK's code seems to set them to 0 too (a bit hard to follow since it uses on channel layout to calc them rather than channel count).
